<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>991</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Team And Task Panel</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="team_table">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>331</width>
      <height>192</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <column>
     <property name="text">
      <string>TeamID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>User ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Project ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="pidlabel">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>20</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>PID</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Team Allocated</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addmemberbtn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>350</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Member</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deletememberbtn">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>350</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Member</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>90</y>
      <width>20</width>
      <height>451</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>90</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Task</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="task_table">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>140</y>
      <width>511</width>
      <height>192</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <column>
     <property name="text">
      <string>Task ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>UserID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Title</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Status</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="addtaskbtn">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>500</y>
      <width>291</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Task</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deletetaskbtn">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>360</y>
      <width>201</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Task</string>
    </property>
   </widget>
   <widget class="QPushButton" name="changetaskstatusbtn">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>400</y>
      <width>201</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Change Task Status</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_completed">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>440</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Completed</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_ongoing">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>470</y>
      <width>95</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Ongoing</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="taskidedit">
    <property name="geometry">
     <rect>
      <x>862</x>
      <y>360</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="placeholderText">
     <string>Task ID</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="titleedit">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>390</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="placeholderText">
     <string>Title</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="enddateedit">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>460</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>440</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Due Date</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="taskdescedit">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>360</y>
      <width>191</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="placeholderText">
     <string>Task Description</string>
    </property>
   </widget>
   <widget class="QPushButton" name="goback">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>460</y>
      <width>281</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Go Back</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="newteamidedit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>390</y>
      <width>281</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="placeholderText">
     <string>Team ID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="newuseridedit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>420</y>
      <width>281</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="placeholderText">
     <string>User ID</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>991</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
